2022/python/Day2.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Dec 25 12:35:49 2022
+
+@author: raegoering
+
+"""
+
+def get_outcome(c1, c2):
+    """
+    Parameters
+    ----------
+    c1 : str
+    c2 : str
+
+    Returns str
+    -------
+    This function evaluates the outcome of a single game of RPS.
+
+    """
+    c1code = {"A": "Rock", "B": "Paper", "C": "Scissors"}
+    c2code = {"X": "Rock", "Y": "Paper", "Z": "Scissors"}
+    losers = [("Rock", "Scissors"), ("Paper", "Rock"), ("Scissors", "Paper")]
+    hand1 = c1code[c1]
+    if c2 in ["X", "Y", "Z"]:
+        hand2 = c2code[c2]
+    else:
+        hand2 = c2
```

Comment:
You assign hand2 to be the original c2 value, I think if you got here, you'd want to error out?